x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/add_exception_flyout/index.tsx

```diff
@@ -493,7 +495,13 @@ export const AddExceptionFlyout = memo(function AddExceptionFlyout({
   }, [listType]);
 
   return (
-    <EuiFlyout size="l" onClose={handleCloseFlyout} data-test-subj="addExceptionFlyout">
+    <EuiFlyout
+      size="l"
+      onClose={handleCloseFlyout}
+      data-test-subj="addExceptionFlyout"
+      // EUI TODO: This z-index override of EuiOverlayMask is a workaround, and ideally should be resolved with a cleaner UI/UX flow long-term
+      maskProps={{ style: `z-index: ${(euiTheme.levels.flyout as number) + 3}` }} // we need this flyout to be above the timeline flyout (which has a z-index of 1002)
```

Comment:
Maybe we could put all `(euiTheme.levels.flyout as number) + #` in a separate place. Maybe be hook which returns all the values or returns a specific one based on passed parameter: `ExceptionFlyout`, `TimelineFlyout` etc...

I'm afraid that it will be hard to track z-index in different places and understand what is an appropriate value if we add a flyout somewhere else.

Reply:
yeah, you're the second person mentioning this. The challenge is this would be used across plugins (cases and security_solution) and I don't really want to create a package just for that... Any idea where would be a good place for this?